simulators/main/oeeAPI.py

```diff
@@ -167,14 +167,22 @@ def add_timeslot(self, locationId, timeslot):
         log.warning(f'Cannot create Timeslot for location:{locationId}, content: {response.status_code} - {response.text}, url: {url}, data: {json.dumps(timeslot)}')
         return False
 
-    def get_shiftplan(self, locationId, dateFrom, dateTo):
+    def get_shiftplan_with_specific_date(self, locationId, dateFrom, dateTo):
         url = f'{self.SHIFTPLAN_REST_ENDPOINT}/{locationId}?dateFrom={dateFrom}&dateTo={dateTo}'
         response = requests.get(url, headers=C8Y_HEADERS)
         if response.ok:
             return response.json()
         log.warning(f'Cannot get shiftplan for {locationId}, url: {url},  response: {response.status_code}: {response.text} ')
         return {'locationId':locationId,'timeslots':{}}
 
+    def get_shiftplan(self, locationId):
+        url = f'{self.SHIFTPLAN_REST_ENDPOINT}/{locationId}'
+        response = requests.get(url, headers=C8Y_HEADERS)
+        if response.ok:
+            return response.json()
+        log.warning(f'Cannot get shiftplan for {locationId}, url: {url},  response: {response.status_code}: {response.text} ')
+        return {'locationId':locationId,'timeslots':{}}
+
```

Comment:
- to avoid code duplication, I'd prefer this method would call `get_shiftplan_with_specific_date(self, locationId, null, null)` and the get_shiftplan_with_specific_date() method handles dates with null values properly